src/index.js

```diff
@@ -1 +1,44 @@
+// eslint-disable-next-line
+// import _ from 'lodash';
+// import inputTable from './modules/inputtable.js';
+// import tableValues from './modules/tableobj.js';
+import { renderContainer, toAddScore } from './modules/createtr.js';
+
 import './style.css';
+
+// inputTable(tableValues);
+
+// let gameIdentity = '';
+// fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+
+//   method: 'POST',
+//   headers: {
+//     'Content-type': 'application/json; charset=UTF-8',
+//   },
+//   body: JSON.stringify({
+//     name: 'Rocking ball',
+//   }),
+// }).then((res) => res.json())
+//   .then((gameID) => {
+//     const response = gameID.result;
+//     gameIdentity = response.substring(14, response.lastIndexOf(' '));
+//     console.log(gameIdentity);
+//   });
```

Comment:
According to [Javascript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) best practices, it is good practice to delete commented code to keep your codebase clean, organized, and readable.

- Kindly delete all commented code from your codebase.